addons/escoria-core/game/core-scripts/esc_game.gd

```diff
@@ -273,3 +286,20 @@ func update_tooltip_following_mouse_position(p_position: Vector2):
 func _set_editor_debug_mode(p_editor_debug_mode: int) -> void:
 	editor_debug_mode = p_editor_debug_mode
 	update()
+
+
+# Pauses the game. Reimplement to eventually show a specific UI.
+func pause_game():
+	pass
+
```

Comment:
There should be another empty line here.

Reply:
Fixed